src/registry.ts

```diff
@@ -135,13 +135,11 @@ export async function getEntry(
     // TODO: Refactor this validation into a separate function.
     /* istanbul ignore next */
     if (!err.response) {
-      console.log(`Full error: ${err}`);
-      throw new Error("Error response not found");
+      throw new Error(`Error response not found. Full error: ${err}`);
```

Comment:
I bumped into this one, it's a bit of a weird error to bump into without context. Can we rephrase it to be more clear as to what we mean here?